icu4c/source/test/intltest/rbbitst.cpp

```diff
@@ -4883,6 +4897,104 @@ void RBBITest::TestDebugRules() {
 #endif
 }
 
+void RBBITest::testTrieStateTable(int32_t numChar, UCPTrieValueWidth expectedTrieWidth,
+                                  int32_t expectedStateRowBits) {
+    // Text are duplicate characters from U+4E00 to U+4FFFF
+    UnicodeString text("");
```

Comment:
Don't need "". The default constructor works fine - 
UnicodeString text;

Reply:
Done